src/Formatter/WhoopsHtmlFormatter.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Equip\Formatter;
+
+use Equip\Adr\PayloadInterface;
+use Whoops\Handler\PrettyPageHandler as HtmlHandler;
+use Whoops\Run as Whoops;
+
+class WhoopsHtmlFormatter extends HtmlFormatter
+{
+    /**
+     * @var Whoops
+     */
+    private $whoops;
+
+    /**
+     * @param Whoops      $whoops
```

Comment:
@ameech I know about it. But in this project I see different styles, so I thought that this is not so important. It would be nice to finally decide.